isolate_example/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```

Comment:
Your `*.dart` files need our copyright header. It's only a few lines, and you can find it in any of the other examples. (If you find an example that's missing it, let me know.)